community/index.html

```diff
@@ -0,0 +1,459 @@
+                                </div>
+                                <div class="grid-item">
+                                    <img src="../media/olol312e/stem.png" class="mb-30" />
+                                </div>
+                                <div class="grid-item">
+                                    <img src="../media/c1omyq2l/tink.png" class="mb-30" />
+                                </div>
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+    </div>
+
+    <footer role="contentinfo" itemscope itemtype="http://schema.org/WPFooter"  id="footer">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 mb-60 col-sm-6">
+                    <h2 class="mb-30">cloudnative.nz</h2>
+                    <p class="lg">Working together to support our communities <br />into pathways in Open Source Cloud</p>
+                    <div class="social mt-60">
+                        <a href="https://www.meetup.com/cloudnative-nz/" target="_blank" class="btn btn-secondary circle mr-30"><i class="fa-brands fa-meetup"></i></a>
+
+                    </div>
+
+                </div>
+                <div class="col-md-4 col-sm-6">
+                    <h3>CONTACT DETAILS</h3>
+                    <p class="lg">
+                        Unit 3, 63 Farm Street<br />
+                        Mount Maunganui<br />
+                        Tauranga<br />
+                        <a href="tel:0220648277">022 064 8277</a>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </footer>
+    <div class="copyright">
+        <div class="container">
+            &copy; Copyright 2022 HE WAKA EKE NOA. All Rights Reserved.
+        </div>
+    </div>
+    <a href="javascript:void(0)" class="scroll-to-top"><i class="fas fa-angle-up" aria-hidden="true"></i></a>
+
+    <div class="modal fade" id="contactModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog modal-lg modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                    <div class="row mt-15 mb-15">
+                        <div class="col-md-6">
+                            <h2>Connect with us <span>Whakapā Mai</span></h2>
+
+                            <div class="inline-row">
+                                <div class="col-3"><i class="fa-solid fa-2x fa-location-dot"></i></div><div class="col-9">
+                                    <p class="lg">
+                                        Unit 3, 63 Farm Street<br />
```

Comment:
Let's update this to either the iimaginerium address or maybe just a email address?

Reply:
Removed address,  set as Tauranga, New Zealand as discussed